File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int n = s.size() - 1;
        int length = 0;

        while(n >= 0 && s[n] == ' ') n-- ;

        while(n>=0 && s[n] != ' ') {
            length++ ;
            n-- ;
        }
        return length;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        n = len(s) - 1
        length = 0

        while (n >= 0 and s[n] == ' ') :
            n-=1
        
        while (n >= 0) and (s[n] != ' ') :
            length+=1
            n-=1
        return length
        